src/components/Button/index.tsx

```diff
@@ -0,0 +1,32 @@
+import styles from './styles.module.scss';
+
+interface Props extends React.ComponentPropsWithoutRef<'button'> {
+  className?: string;
+  submit?: boolean;
+  text: string;
+  variant?: string;
+}
+
+function Button({
+  className = '',
+  disabled = false,
+  submit = false,
+  text,
+  variant = 'contained',
+  onClick
+}: Props) {
+  return (
+    <div>
+      <button
+        className={`${(styles.button, styles[variant])} ${disabled ? styles.disabled : ''} ${className}`}
```

Comment:
Since you are using composes you don't need to specify the base class.

```suggestion
        className={`${styles[variant]} ${disabled ? styles.disabled : ''} ${className}`}
```